plugins/outputs/http/README.md

```diff
@@ -0,0 +1,36 @@
+# HTTP Output Plugin
+
+This plugin writes to a HTTP Server using the `POST Method`
+
+It requires a `url` name.
+
+### Configuration:
+
+```toml
+# Send telegraf metrics to HTTP Server(s)
+[[outputs.graylog]]
```

Comment:
Graylog is my mistake. Based on the output of graylog, I did not care about that part. I will revise that part.